### PR TITLE
fix issue #485

### PR DESCRIPTION
**Pull request by eokodie**

### CODE REVIEW

**Comment by hadley** [comment]
I think you'll also need to do something like `@params ...`.  Currently the documentation is being inherited from `gather()` which is yielding the incorrect advice in the generated docs.

File: R/fill.R [hunk 1]
```diff
@@ -3,14 +3,15 @@ NULL
...
-#' Fills missing values in using the previous entry. This is useful in the
+#' Fills missing values in selected columns using the previous entry. This is useful in the
 #' common output format where values are not repeated, they're recorded
 #' each time they change.
 #'
 #' Missing values are replaced in atomic vectors; `NULL`s are replaced
 #' in list.
 #'
 #' @inheritParams gather
+#' @param ... A selection of columns. If empty, nothing happens. You can supply bare variable names, select all variables between x and z with x:z, exclude y with -y. For more selection options, see the [dplyr::select()] documentation.
```
Comment (hadley) [inline R/fill.R]: This is the right fix, but can you please wrap these lines at ~80 cols? Thanks!

Comment (eokodie) [reply R/fill.R]: Fixes #485

Comment (eokodie) [reply R/fill.R]: Thanks

**Comment by hadley** [comment]
Thank you!

**Pull request merged**